markdown/setup/aqua-data-studio-setup.md

```diff
@@ -0,0 +1,43 @@
+### Aqua Data Studio
```

Comment:
To fellow code reviewers: Please check if this documentation can render all images properly. This is the file to focus for